packages/bruno-js/src/bruno-response.js

```diff
@@ -36,6 +44,10 @@ class BrunoResponse {
     this.body = data;
     this.res.data = data;
   }
+
+  call(expr, ...fns) {
+    return get(this.res.data, expr, ...fns);
```

Comment:
This doesn't satisfy the full feature. lodash.get does not support array filtering.

Reply:
This would be better off using JSONPath instead of lodash.